service/storage/storage.go

```diff
@@ -322,13 +348,17 @@ func (s *service) SyncFileList(ctx context.Context, bucketID string) ([]*models.
 		return nil, err
 	}
 
-	// extract only latest versions; latest version is already sorted
+	// extract only latest versions of files; latest version is already sorted
+	// omit files that don't fit specified created timestamp filtering
 	// on top, add to return list
 	m := map[string]bool{}
 	for _, f := range l {
 		if _, ok := m[f.Name]; !ok {
 			m[f.Name] = true
-			list = append(list, f)
+			if (createdAtSince == nil || (createdAtSince != nil && time.Time(*createdAtSince).Before(time.Time(f.Created)))) &&
```

Comment:
invalid operation: f (variable of type *invalid type) has no field or method Created